src/utils/components/NetworkInterfaceModal/utils/helpers.ts

```diff
@@ -96,7 +96,7 @@ export const createInterface = (
   interfaceType: string,
 ): V1Interface => {
   return {
-    [interfaceType.toLowerCase()]: {},
+    [interfaceType.replace('-', '').toLowerCase()]: {},
```

Comment:
How about defining a mapping between the UI interface Type name (e.g.: `SR-IOV`) and Kubevirt interface API option (e.g: `sriov`)?
It"ll help avoid issues like this PR fixes, also easier to extend in case new type of interface is introduced.

Reply:
Hi @ormergi 
You're right, we already have a mapping but it wasn't used properly, now it should work as you suggested